sources/packages/backend/apps/db-migrations/src/sql/DisbursementOverawards/Add-note-id-added-by-added-date.sql

```diff
@@ -0,0 +1,23 @@
+-- Add column note_id to sims.disbursement_overawards.
+ALTER TABLE
+    sims.disbursement_overawards
+ADD
+    COLUMN IF NOT EXISTS note_id INT REFERENCES sims.notes(id);
+
+COMMENT ON COLUMN sims.disbursement_overawards.note_id IS 'Note id for the disbursement overaward record.';
+
+-- Add column added_by to sims.disbursement_overawards.
+ALTER TABLE
+    sims.disbursement_overawards
+ADD
+    COLUMN IF NOT EXISTS added_by INT REFERENCES sims.users(id);
+
+COMMENT ON COLUMN sims.disbursement_overawards.added_by IS 'User id of the user adding a manual record.';
+
+-- Add column added_date to sims.disbursement_overawards.
+ALTER TABLE
+    sims.disbursement_overawards
+ADD
+    COLUMN IF NOT EXISTS added_date TIMESTAMP WITHOUT TIME ZONE;
```

Comment:
Why is the timestamp without time zone? 

All our assessed_date follow timestamp with time zone.

![image](https://user-images.githubusercontent.com/54600590/222824637-fdf1b666-3353-4721-9129-5bc7adf3fb65.png)

Reply:
OK. You're right. Thanks!